perses/analysis/load_simulations.py

```diff
@@ -2,55 +2,69 @@
-        self.experr = kcal_to_kt(float(self.experr))
-        self.calc = kcal_to_kt(float(self.calc))
-        self.calcerr = kcal_to_kt(float(self.calcerr))
-        self.mw = self.calculate_molecular_weight()
-        self.ha = self.heavy_atom_count()
-        self.simtype = None
-
-    def calculate_molecular_weight(self):
-        """ Calculates the molecular weight of an oemol
-
-        Parameters
-        ----------
 
-        Returns
-        -------
-        float, molecular weight of molecule
-
-        """
-        return oechem.OECalculateMolecularWeight(self.mol)
+import logging
+_logger = logging.getLogger()
+_logger.setLevel(logging.INFO)
+_logger = logging.getLogger("load_simulations")
 
-    def heavy_atom_count(self):
-        """ Counts the number of heavy atoms in an oemol
 
-        Parameters
-        ----------
+class Simulation(object):
+    """Object that holds the results of a free energy simulation.
+    # TODO make this much more flexible
+    Assumes that the output is in the form lig{A}to{B}/out-{phase}.nc
 
+    Parameters
+    ----------
+    A : int
+        integer of first ligand
+    B : int
+        integer of second ligand
 
-        Returns
-        -------
-        int, number of heavy atoms in molecule
-        """
-        return oechem.OECount(self.mol, oechem.OEIsHeavy())
+    Attributes
+    ----------
+    ligA : int
+        integer of first ligand
+    ligB : int
+        integer of first ligand
+    directory : string
+        output directory location : lig{A}to{B}
+    vacdg : float
+        relative free energy in vacuum phase (kcal/mol)
+    vacddg : float
+        variance in relative free energy in vacuum phase (kcal/mol)
+    soldg : float
+        relative free energy in solvent phase (kcal/mol)
+    solddg : float
+        variance in relative free energy in solvent phase (kcal/mol)
+    comdg : float
+        relative free energy in complex phase (kcal/mol)
+    comddg : float
+        variance in relative free energy in complex phase (kcal/mol)
+    vacdg_history : list(float)
+        vacuum free energy at equally spaced intervals of simulation (kcal/mol)
+    soldg_history : type
+        solvent free energy at equally spaced intervals of simulation (kcal/mol)
+    comdg_history : type
+        complex free energy at equally spaced intervals of simulation (kcal/mol)
+    vacddg_history : type
+        variance in vacuum free energy at equally spaced intervals of simulation (kcal/mol)
+    solddg_history : type
+        variance in solvent free energy at equally spaced intervals of simulation (kcal/mol)
+    comddg_history : type
+        variance in complex free energy at equally spaced intervals of simulation (kcal/mol)
+    count : int
+        Number of times the 'histories' have been 'sampled' from
+    hydrationdg : float
+        relative hydration free energy (kcal/mol)
+    hydrationddg : float
+        variance in relative hydration free energy (kcal/mol)
```

Comment:
According to your computation below:
```
            self.hydrationddg = (self.vacddg + self.solddg)**0.5
```
this is the standard error, and *not* the variance.

It might cause less confusion if you always stored the standard error instead?